src/test/java/unit/Group.java

```diff
@@ -106,6 +106,28 @@ void listDescribeUsersReflectCreateDelete() throws IOException {
     matchedGroup =
         groupList.stream().filter(listedGroup -> listedGroup.name.equals(name)).findAny();
     assertTrue(matchedGroup.isEmpty(), "group does not appear in list after delete");
+
+    // create another group
+    String name1 = SamGroups.randomGroupName();
+    TestCommand.runAndParseCommandExpectSuccess(
+        UFGroup.class, "group", "create", "--name=" + name1);
+
+    TestCommand.Result cmd = TestCommand.runCommand("group", "list");
+
+    // use regular expression testing the table format and content inside
+    assertEquals(0, cmd.exitCode, "group list returned successfully");
+    String[] rows = cmd.stdOut.split("\\r?\\n");
+    String[] rowHead = rows[0].split("\\s+");
+    assertEquals(rowHead[0].trim().replace("\r", ""), "EMAIL");
```

Comment:
these should be the other way around, for example `assertEquals("EMAIL", ...)`. The expected value goes before the actual value.